lib/mix/support/migration_helper.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Mix.Support.MigrationHelper do
```

Comment:
These were repeated in a lot of places, so I pulled them out to their own module.